### DISCUSSION TITLE
Legionfancontrol

### DISCUSSION THREAD

**Discussion by kunsh13** [BartoszCichecki/LenovoLegionToolkit#764] [Ideas]
Please check out 
https://www.legionfancontrol.com/ 
and reach out to them and integrate their software in yours

as using this software, we can adjust fan curves even on **non-supported** legion laptops 
(it works on mine and the default vantage app doesn't allow it  
Legion5 15ITH6)

Mabe you can help the guy in https://github.com/BartoszCichecki/LenovoLegionToolkit/discussions/710

**Comment by BartoszCichecki** [comment]
This was discussed many times. LFC will not be integrated with LLT and fan control will not be available on devices prior to gen 7.

**Reply by kunsh13** [reply]
it works on mine tho

**Reply by BartoszCichecki** [reply]
Good, you can keep using it then.

**Reply by kunsh13** [reply]
i wanted one app for all

**Reply by kunsh13** [reply]
and pls tell me which gen is mine

**Reply by BartoszCichecki** [reply]
It's gen 6.

**Reply by kunsh13** [reply]
how do you know?

**Reply by BartoszCichecki** [reply]
Because you wrote which model you have.

**Reply by kunsh13** [reply]
is it the number on the end?

**Reply by BartoszCichecki** [reply]
Yes, the last number is gen. 15ARH5H is gen 5, 16ITH6 is gen 6 and 16ARX8 would be gen 8

**Reply by kunsh13** [reply]
ok thank you
